app-react/src/components/AppMain/AppContainer/AppContainer.js

```diff
@@ -0,0 +1,15 @@
+import style from "./AppContainer.module.css";
```

Comment:
For the react app styling we've been using styled components. I think it would be good to keep things consistent and stick with that (please see `header.jsx` and `footer.jsx`). I think having CSS modules as well as styled components will confuse things. If you could move these styles into styled components that would be great 😄 

I'll update the react app README so this is clear.

Reply:
Yes I understand it will cause confusion. Is there any naming convention you are following for naming styled components? 
The problem I faced is my css class names are same as components name. so it causes naming conflicts. @jrobind

Reply:
There's no hard and fast rules just yet. I think a naming convention like `StyledButton`, `StyledCard` etc should work fine. 

Some of the components we can reuse too, so you might want to define them in a different file rather than within the component file. Totally, up to you – doesn't have to be perfect, as long as we have some consistency with what we're doing.